frontend/src/v5/ui/themes/theme.ts

```diff
@@ -847,24 +856,34 @@ export const theme = createTheme({
 			},
 		},
 		MuiInputLabel: {
+			defaultProps: {
+				shrink: false,
+			},
 			styleOverrides: {
 				root: {
 					position: 'absolute',
 					...typography.body1,
 					fontSize: '12px',
+					top: '-35.5px',
+					left: '-13px',
```

Comment:
![image](https://user-images.githubusercontent.com/62481250/192497087-df91f637-1159-4b62-8488-044a390936e2.png)
This causes problems in the Smart groups settings